backend/app/src/main/java/com/moh/phlat/backend/service/DbUtilityServiceImpl.java

```diff
@@ -220,4 +240,92 @@ public void loadProcessDataToPlr(Long controlTableId, String authenticatedUserId
 		}
 	}
 
+	@Override
+	public List<ReportSummary> getReportSummary(Long controlTableId) {
+		String _attribute;
+		Long _count;
+
+		List<ReportSummary> items = new ArrayList<ReportSummary>();
+
+		_attribute ="TOTAL INPUT RECORDS";
+		_count = processDataRepository.countByControlTableId(controlTableId);
+		ReportSummary rs1 = new ReportSummary();
+		rs1.setAttribute((String) _attribute);
+		rs1.setCount((Long)_count);
+		items.add(rs1);
+		
+		_attribute ="TOTAL INITIAL ROWSTATUS:";
+		_count = processDataRepository.countAllByControlTableIdAndRowstatusCode(controlTableId,"INITIAL");
+		ReportSummary rs2 = new ReportSummary();
+		rs2.setAttribute((String) _attribute);
+		rs2.setCount((Long)_count);
+		items.add(rs2);		
+		
+		_attribute ="TOTAL DO_NOT_LOAD ROWSTATUS:";
```

Comment:
Just a minor thing, can you please remove that `:` from the end?
eg. _attribute ="TOTAL DO_NOT_LOAD ROWSTATUS:"; should be _attribute ="TOTAL DO_NOT_LOAD ROWSTATUS";